OWNERS

```diff
@@ -5,6 +5,7 @@ providers/digitalocean @Deraen
 providers/dnsimple @aeden
 providers/gandi @TomOnTime
 # providers/gcloud
+# providers/hexonet
```

Comment:
If you don't mind putting your name here, it will help you be a recommended reviewer for future issues  / prs referencing this code.

Reply:
I don't mind - it was just part of gettubg "officially" supported.